desc/protoparse/reporting_test.go

```diff
@@ -58,7 +58,7 @@ func TestErrorReporting(t *testing.T) {
 				"test.proto:5:41: syntax error: unexpected \"enum\", expecting ';'",
 				"test.proto:5:69: syntax error: unexpected ';', expecting '='",
 				"test.proto:7:53: syntax error: unexpected '='",
-				`test.proto:2:50: syntax value must be "proto2" or "proto3"`,
+				`test.proto:2:50: syntax value must be "proto2" or "proto3" but was "proto"`,
```

Comment:
We might actually not want to change these error messages, this was probably overzealous on my part